scripts/build-dist.ps1

```diff
@@ -14,6 +14,4 @@ tar -xzf pack.tgz --strip 1
 rm pack.tgz
 # Change this to "yarn install --production" once #1115 is fixed
 npm install --production
-rm node_modules/*/test -Recurse
-rm node_modules/*/dist -Recurse
```

Comment:
These rules aren't up to date so I removed them, not sure where this is actually used. @Daniel15 can you advise?